### PR TITLE
attempted to make a random data generator for task a emulation

### PR DESCRIPTION
**Pull request by peterphilips**
I attempted to make some random data generation into a function. It seems to pass the rdftool validation but please double check

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+    # FOI
+    # need to keep track of list for code below
+    foi_list = []
+    for index in range(0, sampling_number(n)):
+        foi = URIRef("https://linked.data.gov.au/dataset/bdr/site/" + samplings_data[index]["foi"])
+        # + "/foi_number/" + str(index + 1))
+        foi_list.append(foi)
+        g.add((foi, RDF.type, TERN.FeatureOfInterest))
+        g.add((foi, TERN.featureType, Literal(random.choice(FEATURE_TYPES))))
```
Comment (nicholascar) [inline utils.py:103]: You can't do this: the range value of TERN.featureType (see https://w3id.org/tern/ontologies/tern/FeatureOfInterest) is not a literal but an IRI, so you have to turn all the values you have for FEATURE_TYPES (and also METHOD_TYPES) into IRI. Do this with a list of URIRefs of values from the TERN vocabs (https://linkeddata.tern.org.au/viewer/tern/) which you know, like this:

```python
FEATURE_TYPES = [
    URIRef("http://linked.data.gov.au/def/tern-cv/05dac53a-269c-4699-9673-bf99a9406b14"),  # administrative area
    URIRef("http://linked.data.gov.au/def/tern-cv/ecb855ed-50e1-4299-8491-861759ef40b7"), # animal individual
    ...
]
```

and then add the IRI triple:

```python
g.add((foi, TERN.featureType, random.choice(FEATURE_TYPES)))
```

We don't actually care about particular values or labels, so just the opaque IRIs are fine

**Comment by nicholascar** [comment]
Just the one change needed then I'll merge

**Comment by peterphilips** [comment]
@nicholascar Refer to 91cd9e5b07bd1f84f1daa4263c88ad5b2a020b4f and see if that's what was intended

**Comment by nicholascar** [comment]
Yes, that's correct but you will have to do the same for `METHOD_TYPES` too

**Comment by peterphilips** [comment]
@nicholascar Yep see: d6796c7697ff53f0a42fc87b3dbeb2cb0f727c46 note i also added the outputted file and validated according to BDR messages, geosparql, and tern shapes. all successful :)

**Comment by nicholascar** [comment]
Great, good to go!

**Pull request merged**